portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,155 @@
+                <p>Throughout high school, I was involved in several extracurriculars- including teaching students using Google CS First, volunteering at the Liberty Science Center, and working with our school's chapter of the National Honor Society to help out with community events. I am glad that I was able to give back to my community when I had the chance. </p>
+                <p>After high school, I was unsure about whether I wanted to pursue a career in computer science or business. Computer science - I was interested in but scared to pursue. Business - I hated, but it was a safe option. Luckily, I was accepted into Google CSSI, where I met 29 other very accomplished students with similar goals and ambitions. In a short 3 weeks, we became unexpectedly close. To this day, we hang out and even attend hackathons together. </p>
+                <p>In college, I have become an active leader in the Honors Academy in the School of Engineering. I help interview perspective students and run panels to answer FAQ's. I love to get involved, and I'm glad Google SPS will be a step in the right direction. </p>
+             </div>
+
+             <div class="card">
+                <h2>What I enjoy to photograph</h2>
+                <p>I love to photograph anything that can trace back to a memory. Here are some pictures I have taken. </p>
+                <div class="row"> 
+
+                    <div class="column">
+                        <img src="/images/1.png" style="width:100%">
+                        <img src="/images/2.png" style="width:100%">
+                        <img src="/images/4.png" style="width:100%">
+                        <img src="/images/9.png" style="width:100%">
+                    </div>
+
+                    <div class="column">
+                        <img src="/images/3.png" style="width:100%">
+                        <img src="/images/8.png" style="width:100%">
+                    </div>  
+
+                    <div class="column">
+                        <img src="/images/5.png" style="width:100%">
+                        <img src="/images/6.png" style="width:100%">
+                        <img src="/images/7.png" style="width:100%">
+                    </div>
+
+                </div>
+
+                </div>
+        </div>
+
+        <div class="rightcolumn">
+
+            <div class="card">
+                <button class="button" onclick="addRandomGreeting()">Click for a fun fact about me</button>
+                <br>
+                <br>
+                <div id="greeting-container"></div>
+            </div>
+
+            <div class="card">
+                <h3>Past Projects</h3>
+                <ul>
+                    <li> <a href="http://clarity-support.appspot.com/">Clarity</a> a site for self esteem.</li>
```

Comment:
Looks like you have prior App Engine experience? From CSSI I'm guessing?

Reply:
Yup, we used app engine for two projects then.